src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -74,6 +72,7 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
 
   const investedAmount = investFlowData[optionIndex].investedAmount
   const inputError = investFlowData[optionIndex].error
+  const isTouched = investFlowData[optionIndex].isTouched
```

Comment:
I'm confused. Anxo suggested to not use redux and you said you are not anymore, but here you still are.

Am I viewing outdated code?

Reply:
A yea, reverted back to redux because I need that data out of that component for the InvestFlowValidation message. With that revert removed the comment will add it back!